src/Transports.Subscriptions.WebSockets/WebSocketConnection.cs

```diff
@@ -18,9 +18,16 @@ public class WebSocketConnection
 
         public async Task Connect()
         {
-            await _server.OnConnect();
-            await _server.OnDisconnect();
-            await _transport.CloseAsync();
+            try
+            {
+                await _server.OnConnect();
+                await _server.OnDisconnect();
+                await _transport.CloseAsync();
+            }
+            finally
+            {
+                _transport.Dispose();
```

Comment:
Dispose transport? Why?

Reply:
`WebSocketConnection` should implement `IDisposable` and dispose `_transport`.

Reply:
Even CodeQL shows warning above.

Reply:
Already in PR #673 but if the code doesn't work now, it won't work with that change.